src/i18n/en/data/dmarc.ts

```diff
@@ -0,0 +1,9 @@
+export default {
+    v: "This is commonly at the start of a record name to define the version. This is commonly equal to DMARC1.",
```

Comment:
I don't like the use of `Commonly` here, makes it feel very informal.
This should read something like "This dictates the DMARC protocol version that is being used. Normally located at the start of a record and set to DMARC1"?